SharedModules/endfConstants.f90

```diff
@@ -110,6 +112,8 @@ module endfConstants
   integer(shortInt), parameter :: macroAllScatter = -20 ,&
                                   macroAbsorbtion = -21 ,&
                                   noInteraction   = -901
+  ! List of Macro Energy Deposition numbers for different energy deposition modes. Unique to SCONE
+  integer(shortInt), parameter :: macroEnergyDepoZero = -1100
```

Comment:
I don't think there needs to be a number for mode 0 in particular at this stage. Rather, just energy deposition (or kappaFission) is fine, sitting with the other macros for consistency of style. Also, I might suggest making it -80 for consistency with Serpent. The micro would then be -301.
https://serpent.vtt.fi/mediawiki/index.php?title=ENDF_reaction_MT%27s_and_macroscopic_reaction_numbers